code/modules/unit_tests/spawn_mobs.dm

```diff
@@ -0,0 +1,8 @@
+///Unit test that spawns all mobs that can be spawned by golden slimes
+/datum/unit_test/spawn_mobs
+
+/datum/unit_test/spawn_mobs/Run()
+	for(var/T in typesof(/mob/living/simple_animal))
```

Comment:
Please change this to _animal, I dislike single letter variable names like this (especially when they're used for more than just a type cast).

Reply:
Yeah I must admit I kinda copied the for from mob.dm and forgot to change the var name, fixed now